### ISSUE TITLE
Slowness to load

### ISSUE THREAD

**Issue by margitZ** [isf-hack-night/reps-app#2]
Known issue: Loading each district outline seperately - how to optimize (cc @8gian )
Possible Unknown:  Unexplained performance degradation experience by @RS-Crawford and @margitZ

**Comment by RS-Crawford** [comment]
Yeah this is a huge one in my opinion. I think there's a number of ways to optimize the performance here. Some thoughts:


1.  Prioritize districts based on population
  - We know the majority of people in California live in dense urban areas like the Bay and LA, we could prioritize those districts first
2.  Use some kind of hash check first
  - If we chunk California (let's say vertically) we can make a quick check of "are you in Cali and between these two lats?"
  - If a user is, we can load a certain smaller number of potential districts
3. With 2 districts in query params (after the home page) we can grab those 2 districts they're in right away and then load others in the background
4. Have this data loaded more appropriately within WordPress server rather than as-is with media files

... and others? Whatever else  y'all can come up with.

**Comment by margitZ** [comment]
Map is definitely getting initialized twice.

<img width="1306" alt="screen shot 2017-08-30 at 9 46 01 pm" src="https://user-images.githubusercontent.com/1396362/29906848-b8e17e4e-8dcc-11e7-8f86-0ab501e84d59.png">

**Comment by margitZ** [comment]
For dev purposes, it helps MASSIVELY to uncheck 'Disable Cache' on performance tab in dev tools.  (not sure why I had this turned off in the first place, unless it had something to do with the injection)

**Comment by margitZ** [comment]
If clear cache, initial load is still really slow, but district switching performance seems fine.

**Comment by margitZ** [comment]
@black98fxstc is working on this issue.
